### ISSUE TITLE
Unable to open options once plugin installed

### ISSUE THREAD

**Issue by tinmice** [benji300/joplin-note-tabs#46]
Once I have installed the plugin, I am no longer able to open the main options menu.

```
Error
Joplin encountered a fatal error and could not continue. To report the error, please copy the *entire content* of this page and post it on Joplin forum or GitHub.

Message
md.options is not a function

Version info
Joplin for Desktop

Copyright © 2016-2021 Laurent Cozic
Joplin 1.6.8 (prod, win32)

Client ID: e02e9d22a03e4d7085f817885bcc1a9f
Sync Version: 2
Profile Version: 34
Keychain Supported: Yes

Revision: fc7bd41c1 (master)
Stack trace
TypeError: md.options is not a function
    at ConfigScreenComponent.settingToComponent (C:\Program Files\Joplin\resources\app.asar\gui\ConfigScreen\ConfigScreen.js:331:39)
    at createSettingComponents (C:\Program Files\Joplin\resources\app.asar\gui\ConfigScreen\ConfigScreen.js:144:42)
    at ConfigScreenComponent.sectionToComponent (C:\Program Files\Joplin\resources\app.asar\gui\ConfigScreen\ConfigScreen.js:149:30)
    at Object.shared.settingsToComponents2 (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\components\shared\config-shared.js:184:28)
    at ConfigScreenComponent.render (C:\Program Files\Joplin\resources\app.asar\gui\ConfigScreen\ConfigScreen.js:528:37)
    at finishClassComponent (C:\Program Files\Joplin\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:17039:31)
    at updateClassComponent (C:\Program Files\Joplin\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:16994:24)
    at beginWork$1 (C:\Program Files\Joplin\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:18505:16)
    at HTMLUnknownElement.callCallback (C:\Program Files\Joplin\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:347:14)
    at Object.invokeGuardedCallbackDev (C:\Program Files\Joplin\resources\app.asar\node_modules\react-dom\cjs\react-dom.development.js:397:16)
Component stack

    in ConfigScreenComponent (created by Connect(ConfigScreenComponent))
    in Connect(ConfigScreenComponent) (created by NavigatorComponent)
    in div (created by NavigatorComponent)
    in NavigatorComponent (created by Connect(NavigatorComponent))
    in Connect(NavigatorComponent) (created by RootComponent)
    in ThemeProvider (created by RootComponent)
    in StyleSheetManager (created by RootComponent)
    in RootComponent (created by Connect(RootComponent))
    in Connect(RootComponent)
    in ErrorBoundary
    in Provider
```

**Comment by benji300** [comment]
Which version of the plugin have you tried to install?
Can you reproduce the issue or was it just a one-time error?
Are you sure that this issue comes from this plugin? Maybe you installed another plugin which is not supported with Joplin version 1.6.8?

**Comment by wolcen** [comment]
I'm also having this issue. Given I installed the plugin from the options screen I know that it was just working and stopped immediately after this install. I installed via search for `tabs` in the plugins screen, as directed for >v1.6.4. Did not note what version that installed.

I'm on Joplin 1.6.7 (prod, linux)

**Comment by wolcen** [comment]
And no - not a one-time crash. Restart, do nothing, and just go to options and it crashes. No other plugins installed - this is the first time I've tried one. I tried unpinning the one pinned tab I had, but that made no difference.

**Comment by wolcen** [comment]
OK - had no idea I was so outdated - I've updated to v1.8.5 and no longer have this issue. FWIW, the plugin version is v1.3.0

**Comment by benji300** [comment]
With version v1.3.0 I've added some dropdown menu options. I guess these new options are responsible for this issue. Honestly, I thought they would also work with the specified minimal app version. 
I will have a look at it. It will come down to a new minimal app version though (I think).
So as a workaround, I recommend switching to a new app version or an older plugin version.

**Comment by benji300** [comment]
Fixed with latest release v1.4.0 (#50).
I've updated the required minimum version for the plugin to v1.8.2. So this issue should be "fixed".
